src/Security/MemberAuthenticator/ChangePasswordHandler.php

```diff
@@ -144,23 +138,6 @@ public function changepassword()
         );
     }
 
-
-    /**
-     * @param Member $member
-     * @param string $token
-     */
-    protected function setSessionToken($member, $token)
```

Comment:
We can't remove `protected` functions in a minor release, as they count as part of our "Public API" -  see https://docs.silverstripe.org/en/5/project_governance/public_api/